english/explore-outside/journey-to-flaming-tower/jellybus/jellybus.md

```diff
@@ -1,5 +1,7 @@
 At the nearby jellybus stop you see that the next jellybus will be arriving in
-15 Chocolate Kisses. There is another individual waiting there too.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.